velox/dwio/dwrf/test/ColumnWriterTest.cpp

```diff
@@ -1233,16 +1238,16 @@ TEST_F(ColumnWriterTest, TestMapWriterFloatKey) {
 
   EXPECT_THROW(
       {
-        testMapWriterNumericKey<float>(
-            /* useFlatMap */ true, /* useStruct */ true);
+        testMapWriterNumericKeyUseStruct<float>(
+            /* useFlatMap */ true);
       },
       exception::LoggedException);
 }
 
 TEST_F(ColumnWriterTest, TestMapWriterInt64Key) {
   testMapWriterNumericKey<int64_t>(/* useFlatMap */ false);
   testMapWriterNumericKey<int64_t>(/* useFlatMap */ true);
-  testMapWriterNumericKey<int64_t>(/* useFlatMap */ true, /* useStruct */ true);
+  testMapWriterNumericKey<int64_t, /* useStruct */ true>(/* useFlatMap */ true);
```

Comment:
nit: maybe use `testMapWriterNumericKeyUseStruct` since we added that and it is easier to read.